src/messages/GE.json

```diff
@@ -334,7 +334,7 @@
   "Sidebar minimized": "Sidebar minimized",
   "Sidebar maximized": "Sidebar maximized",
   "Response area expanded": "Response area expanded",
-  "Close expanded response area": "Close expanded response area"
-  
-}
-
+  "Close expanded response area": "Close expanded response area",
+  "permissions not found in permissions tab": "Permissions for the query are missing on this tab, use Select permissions option in the settings icon next to your profile to see the full list of Microsoft Graph permissions and select the permission(s) you want and consent to them from there.",
```

Comment:
This message may not apply to someone who is not logged in since they would not be able to see that button. How about splitting this message further so that it can be displayed on the condition that someone is logged in? Or having the message explain that they'd have to log in?

Reply:
Good point. I will create a separate message for a user that isn't logged in.